examples/CloudRetro/README.md

```diff
@@ -0,0 +1,509 @@
+* (ln. 65) worker -> network -> publicAddress: This should be your <YOUR_W_EXTERNAL_IP>
+* (ln. 65) worker -> server -> address: Should be the TCP port youve used during the worker deployment (9000 in this example)
+
+After overwriting and saving these values, you will need to restart the deployments for this changes to take effect:
+
+```console
+kubectl rollout restart deployment -n cloudretro worker-deployment
+kubectl rollout restart deployment -n cloudretro coordinator-deployment
+```
+
+Now, if you connect to the coodinator HTTPs, you should be welcomed with the same screen. But if you check your worker list with the little w button on the top-left corner of the console, you should see your workers.
+Congratulations, you`ve (almost) set up the CloudRetro on your cluster!
+
+If you check your browsers console, you could be seeing an ICE-candidate exchange, which ultimately results in fail. That is because if we take a look at the worker-pod log the client is trying to build a connection with, only generates a host type Ice candidate, which is unreachable for the client due to the massive amounts of NATs its probably behind of. This is when STUNner comes into picture.
+
+
+
+### Installing STUNner
+
+STUNner is a Kubernetes ingress gateway for WebRTC, it exposes a standards-compliant STUN/TURN gateway for clients to access your virtualized WebRTC infrastructure running in Kubernetes.
+We will be using STUNner to forward our WebRTC media stream.
+
+First, we will deploy STUNner as a turn server, and configure it, then create a configurable Gateway for it, so we can forward an UDP Route through that. And all this inside the cluster. Marvelous.
+
+First of all, create a namespace for stunner:
+
+```console
+kubectl create namespace stunner
+```
+
+#### Installing STUNner gateway operator
+
+This will be a bit tricky and unlogical, but it wil be fixed asap.
+
+Clone the STUNner gateway operator git repo and enter into the root directory:
+
+```console
+git clone https://github.com/l7mp/stunner-gateway-operator.git
+cd stunner-gateway-operator
+```
+
+Install the Kubernetes Gateway CRDs from the official source (these are not part of the STUNner distribution).
+
+```console
+kubectl apply -k "github.com/kubernetes-sigs/gateway-api/config/crd?ref=v0.4.3"
+```
+
+Deploy the STUNner Kubernetes Gateway Operator:
+
+```console
+make install
+make deploy
+```
+
+Confirm the operator is running in stunner-gateway namespace:
+
+```console
+kubectl get pods -n stunner-gateway-operator-system
+
+NAME                                                          READY   STATUS    RESTARTS   AGE
+stunner-gateway-operator-controller-manager-65dbf8fb4-hjrjr   2/2     Running   0          42m
+```
+
+#### Configuring the operator
+
+The STUNner operator (partially) implements the official Kubernetes Gateway API, which allows you to interact with STUNner using the convenience of kubectl and declarative YAML configuration.
```

Comment:
This is the super-interesting stuff. Please don't explain what GatewayClass/GatewayConfig/... objects are here, just refer to the gateway-operator docs. The idea here is to explain how we use these resources to expose cloudretro, so just include as much context here as needed for the reader to undersrand that.

Once superimportant thing to describe here is: what is the thingie that we are exposing here? The worker? the coordinator? why are we useing this? how do we expose the coordinator? etc.